src/core/Scene.ts

```diff
@@ -1189,6 +1380,8 @@ export class Scene extends SharedAPI {
 		${super.css}
 
 		:host {
+			pointer-events: auto;
```

Comment:
- [ ] I don't remember why we need to add this (or I forgot to remove it when experimenting). Let's add a comment, or remove if not needed.

Reply:
I saw this a while ago and don't know why. I know we messed with disabling pointer-events for CSS click events and maybe this was an artifact of that